app/src/main/java/com/xmartlabs/daydreaming/ui/CustomScreenFragment.java

```diff
@@ -21,12 +24,16 @@
  */
 @FragmentWithArgs
 public class CustomScreenFragment extends BaseFragment {
+  public static final int MAX = 100;
```

Comment:
But it's only used in `moodSeekBarView.setMax(MAX);`. Don't you think it is clear enough?